WeScan/Scan/CaptureSessionManager.swift

```diff
@@ -122,6 +128,18 @@ final class CaptureSessionManager: NSObject, AVCaptureVideoDataOutputSampleBuffe
     }
     
     internal func capturePhoto() {
+        
+        let captureConnection = photoOutput.connections.first { (connection) -> Bool in
+            return connection.inputPorts.first(where: { (port) -> Bool in
+                port.mediaType == .video
```

Comment:
Why are we matching for `video` here while we're taking a photo?

Reply:
Because we need connection input port (video camera) to collect each frame image for the user to display.